nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss

```diff
@@ -136,6 +136,10 @@
         flex-basis: 100%;
         flex: 1;
     }
+
+    .mat-mdc-paginator {
+        background-color: unset !important;
```

Comment:
I think we likely want to just set it to the mat-app-background-color:

```suggestion
        background-color: var(--mat-app-background-color) !important;
```

Reply:
That assumes we always want to put the paginator on our app background. setting is as `unset` allows it to be used on any background.

Reply:
Fair enough. Let's roll.